src/Access/AccessControl.cpp

```diff
@@ -447,8 +447,13 @@ UUID AccessControl::authenticate(const Credentials & credentials, const Poco::Ne
 {
     try
     {
-        return MultipleAccessStorage::authenticate(credentials, address, *external_authenticators, allow_no_password,
+        LOG_DEBUG(getLogger(), "AccessControl::authenticate {} user {}",
+                typeid(credentials).name(), credentials.getUserName());
```

Comment:
Isn't it enough logging for authentication already?

Reply:
Those are leftovers from debugging session and should be removed